pkg/webhook/config/helper.go

```diff
@@ -0,0 +1,257 @@
+/*
+   Copyright 2017 The Kubernetes Authors.
```

Comment:
this file is not used?

Reply:
It is used by `pkg/webhook/config/validating.go`.

Reply:
oh, i see.